inscriptions/ISic000546.xml

```diff
@@ -187,42 +192,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR138302" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la">
                 <ab>
-                    <lb n="0"/>
-                    <gap reason="lost" extent="unknown" unit="line"/>
-                    <lb n="1"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <expan>
-                        <abbr>Q</abbr>
-                        <ex>uinti</ex>
-                    </expan>
-                    Fabi Caesili Titiani
-                    <lb n="2"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>avo
-                    <expan>
-                        <abbr>Q</abbr>
-                        <ex>uinto</ex>
-                    </expan>
-                    Aquilio Nigro
-                    <lb n="3"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost">p</supplied>roco
-                    <expan>
-                        <ex>n</ex>
-                        <abbr>s</abbr>
-                        <ex>uli</ex>
-                    </expan>
-                    <lb n="4"/>Si<supplied reason="lost">cili</supplied>ae.
+                    <lb n="0"/><gap reason="lost" extent="unknown" unit="line"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">et</supplied> <expan><abbr>Q</abbr><ex>uinti</ex></expan> <g ref="#interpunct">·</g> Fabi <g ref="#interpunct">·</g> Caesili <g ref="#interpunct">·</g> Titiani
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/>avo <g ref="#interpunct">·</g> <expan><abbr>Q</abbr><ex>uinto</ex></expan> <g ref="#interpunct">·</g> Aquilio <g ref="#interpunct">·</g> Nigro
+                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">p</supplied>ro <g ref="#interpunct">·</g> <expan><abbr>co</abbr><ex>n</ex><abbr>s</abbr><ex>uli</ex></expan> <!--unsure if I have coded this correctly, wanted to put co(n)s(uli)-->
+                    <lb n="4"/>Si<supplied reason="lost">cili</supplied>ae
                     <lb n="5"/>
                 </ab>
                 <!--This EDR text 138302 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
             </div>
             <div type="apparatus">
                 <p><!--external apparatus criticus (if applicable)--></p>
```

Comment:
replace: `<p><!--external apparatus criticus (if applicable)--></p>`
with: `<listApp><app><note>Text of CIL</note></app></listApp>`